algrtm/src/components/DataLabSection.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import dataLabSectionTexts from "@/data/dataLabSectionTexts";
+
+const DataLabSection = () => {
+  return (
+    <section className="w-full flex flex-col gap-6 mb-[2.75rem] lg:my-[5.88rem] mt-[2.75rem] mx-[1rem] md:mt-[7.75rem] md:mx-[4.56rem] xl:mx-[5.88rem] bg-neutralLight-neutral100 dark:bg-neutralDark-neutral100">
+      <div className="h4 lg:h2SB xl:h2 mx-[1rem] md:mx-[4.56rem] xl:mx-[5.88rem] text-neutralLight-neutral10 dark:text-neutralDark-neutral10">
+        Our Data Laboratory
+      </div>
```

Comment:
This text moves to the left, while the cards remain centered, so it looks weird on higher resolutions

![image](https://github.com/Nospoko/algrtm/assets/26410548/cc6eea78-b85e-482c-b63b-855d09c5a666)

We can try to limit the width of the div with the text to be aligned with the first card

Reply:
I suggest to center the title too, looks much better to me, write me what you think:)

Reply:
Honestly, both look good to me 😄